presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -266,7 +268,7 @@ else if (typeSignature.startsWith("varchar")) {
             }
 
             // Use writeOrcColumnsPresto so that orcType and varchar length can be written in file footer
-            writeOrcColumnsPresto(orcFile, ORC_12, NONE, Optional.empty(), Collections.nCopies(channelCount, type), values, new OrcWriterStats());
+            writeOrcColumnsPresto(orcFile, DWRF, NONE, Optional.empty(), Collections.nCopies(channelCount, type), values, new OrcWriterStats());
```

Comment:
Why is this change? It needs to go in a separate commit if it is needed.

Reply:
It was a leftover from testing integer types and it's not related to byte and boolean. I have removed the change.